### ISSUE TITLE
Remove comments within production *.html files in dist directory

### ISSUE THREAD

**Issue by mischah** [micromata/bootstrap-kickstart#28]
Comments like the following doesn’t need to be in the production html files:
```html
<!--
  Replace "no-js" class with "js" in case JavaScript is available.
  You can delete this when you feel the need to make use of modernizr.
-->
```

@krnlde @mfroehner 
I think this is a no-brainer. Thanks for the suggestion :blush: 

But what to you think about going one step further and use [HTMLMinifier](https://github.com/kangax/html-minifier) to get rid of comments as well as whitespaces and line breaks?

Pros:
- Save a few kilobytes

Cons:
- Inspecting the processed html files within an editor is useless
 - In the browser view in the Dev Tools aren’t affected.
- Might cause problems in oldIEs because I remember browser bugs which are caused by whitespace 
 - But as long as I know several bugs are caused by whitespace and can be »fixed« by removing whitespace.

What’s your opinion on that?

**Comment by krnlde** [comment]
I wouldn't want to see that in production. The savings are not enough to back up the mess you create when opening the source code. GZIP is applied to HTML files, so the kilobytes mentioned vanish anyway. Since the tags and attributes cannot be shortened in anyway the savings you get from minification reduce just to whitespaces and line breaks.

TL;DR no I don't like HTML minification.

**Comment by mischah** [comment]
Thanks for your feedback.

> TL;DR no I don't like HTML minification.

I guess I'm with you :blush:

**Comment by mischah** [comment]
Done. Will make it to the next release.
